deploy/instance/local-scripts/sync.sh

```
@@ -0,0 +1,39 @@
+REMOTE_HOST="root@saf-demo.online"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)